go.mod

```diff
@@ -3,22 +3,32 @@ module github.com/Trendyol/kafka-cronsumer
 go 1.19
 
 require (
+	github.com/prometheus/client_golang v1.16.0
 	github.com/robfig/cron/v3 v3.0.1
 	github.com/segmentio/kafka-go v0.4.39
 	go.uber.org/zap v1.24.0
 )
 
 require (
+	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/cespare/xxhash/v2 v2.2.0 // indirect
+	github.com/golang/protobuf v1.5.3 // indirect
```

Comment:
This comes from client_golang, right?

Reply:
yes it is coming from github.com/prometheus/client_golang